src/appsco-manage-customer-page.html

```diff
@@ -301,12 +354,8 @@
                 }
             },
 
-            _computeLogApi: function(customer) {
-                return customer.meta ? customer.meta.log : '';
-            },
-
-            _computeGroupsApi: function(customer) {
-                return customer.meta ? customer.meta.groups + '?extended=1' : '';
+            _computePartnerAdminsApi: function(customer) {
+                return (customer && customer.meta) ? customer.meta.list_partner_admins : null;
```

Comment:
Should be checked if 'null' will cause iron-ajax to load. I'm sure that empty string will not trigger AJAX load.